.gitignore

```diff
@@ -25,6 +25,7 @@
 
 # Visual Studio cache directory
 .vs/
+.vsconfig
```

Comment:
This is really nitpicky, but I think it's better if this line is moved to below the `# Custom patterns` comment at the bottom of the file, because everything above that comment has been generated from [gitignore.io](https://gitignore.io/), and it's useful to be able to update that part of the file without having to account for custom patterns that we added 😅